### DISCUSSION TITLE
Calamares installer (Budgie variant) fails during install w/ LUKS

### DISCUSSION THREAD

**Discussion by trharmon0** [SpiralLinux/SpiralLinux-project#510] [Q&A]
I'm trying to install to /dev/nvme1n1 (see below) but the installer fails regardless of whether I try to install the boot loader to this drive or any of the others and also regardless of whether I ask for swap or swap + hibernate. For all attempts, I am specifying to install into LUKS.

```
sda       8:0    0   4.5T  0 disk 
└─sda1    8:1    0   4.5T  0 part 
sdb       8:16   0 931.5G  0 disk 
└─sdb1    8:17   0 931.5G  0 part 
sdc       8:32   1  14.6G  0 disk 
├─sdc1    8:33   1   2.1G  0 part /usr/lib/live/mount/medium
│                                 /run/live/medium
└─sdc2    8:34   1     5M  0 part 
sdd       8:48   1     0B  0 disk 
zram0   253:0    0 119.4G  0 disk [SWAP]
nvme1n1 259:0    0   3.6T  0 disk 
nvme0n1 259:1    0   1.8T  0 disk 
```

The error on the installer is:

```
The installer failed to create partition on disk 'Sabrent Rocket Q'.

Details:
Create a new partition (3.64 TiB, luks) on /dev/nvme1n1'
```

The really odd part is that I **just did** an install to this drive a few days ago with no hiccups whatsoever.

All of the drives report clean from smartmontools.

Thoughts?

**Comment by trharmon0** [comment]
Same error even if I **do not** ask for LUKS. And though the disk's part label was set to GPT it shows as 'msdos' after the error.

**Reply by marwenius** [reply]
Maybe it is a mix of MSDOS and GPT? This could cause the Calamares installer to fail (unrelated to LUKS).

What is the output of: `sudo fdisk -l`? (lower case L)

**Reply by trharmon0** [reply]
Unfortunately I have since over-installed the system and cannot inspect. That said, before each run of the installer I would boot to Live CD and make sure that all drives were using GPT. After any attempt to install the target drive would end up being labeled as MS-DOS format. Very odd.

Given that I won't be testing this again soon I will Close and it can be here if anyone runs into this issue later. Or if I return to using Spriral in this context. I hope to do so because the automatic set up of snapshots is pretty compelling.

thx
